assets/js/components/ProjectEditor.js

```diff
@@ -204,6 +214,39 @@ export function ProjectEditor (projectDescriptionCredits, programId, textFields,
     }
   }
 
+  this.deleteTranslation = () => {
+    const languageSelected = this.languageSelect.value
+    if (languageSelected === '' || languageSelected === 'default') {
+      showSnackbar('#share-snackbar', this.cannotDelete)
```

Comment:
With this change, if the user tries to delete the default name, description, and credits then the delete request will not go through and a snackbar is shown telling the user that the default cannot be deleted.

However, in the next PR, I will be reworking the navigation based on the following document which will make this logic out of date. So, in a couple of weeks, I will make a PR that will hide or disable all the delete, create, and save buttons on this page. But the current logic should be okay for now and will not cause any problems.